docs/management/api/blocklist-api.asciidoc

```diff
@@ -0,0 +1,614 @@
+:endpoint-artifact-list-id:                     endpoint_blocklists
+:endpoint-artifact-name:                        blocklist entries
+:endpoint-artifact-name-single:                 blocklist entry
+:endpoint-artifact-name-sentence-start:         Blocklist entries
+:endpoint-artifact-name-single-sentence-start:  Blocklist entry
+:endpoint-artifact-api-doc-name:                blocklist
+[[blocklist-api]]
+=== {endpoint-artifact-name-sentence-start}
+
+Create, retrieve, update, and delete endpoint <<blocklist>> entries with the <<exceptions-api-overview>>. Endpoint {endpoint-artifact-name} are managed using a static container id (`list_id`) of `pass:a[{endpoint-artifact-list-id}]`, which must be created prior to adding the {endpoint-artifact-name}. Access to these APIs requires that a user has authorization to manage endpoints.
+
+
+NOTE: Console supports Elasticsearch APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Run Elasticsearch API requests].
+
+
+==== Create {endpoint-artifact-name} container
+
+`POST <kibana host>:<port>/api/exception_lists`
+
+===== Request body
+
+A JSON object containing the fields listed below. The request must have the following:
+
+include::_exceptions-api-reusable-content.asciidoc[tags=endpoint-exceptions-request-payload-list-id-and-namespace-requirements]
+- The `type` value must be `endpoint`.
+
+include::_exceptions-api-reusable-content.asciidoc[tags=create-exception-container-request-body]
+
+===== Example request
+
+[source,console,subs="attributes"]
+--------------------------------------------------
+POST api/exception_lists
+{
+  "description": "Endpoint Security Blocklists List",
+  "name": "Endpoint Security Blocklists List",
+  "list_id": "{endpoint-artifact-list-id}",
+  "type": "endpoint_blocklists",
+  "namespace_type": "agnostic"
+}
+--------------------------------------------------
+// KIBANA
+
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+===== Response payload
+
+[source,json,subs="attributes"]
+--------------------------------------------------
+{
+  "_tags": [],
+  "created_at": "2020-07-13T09:33:46.187Z",
+  "created_by": "elastic",
+  "description": "Endpoint Security Blocklists List",
+  "name": "Endpoint Security Blocklists List",
+  "list_id": "{endpoint-artifact-list-id}",
+  "type": "endpoint_blocklists",
+  "namespace_type": "agnostic",
+  "id": "f320c070-c4eb-11ea-80bb-11861bae2798",
+  "tags": [],
+  "tie_breaker_id": "2c08d5a5-2ecc-4d5a-acfb-0a367f25b3f3",
+  "updated_at": "2020-07-13T09:33:46.359Z",
+  "updated_by": "elastic"
+}
```

Comment:
There are some missing response bits here (`immutable`, `os_types`, `_version`, `version`), I notice they need to update in all other artifacts as well except for in Host Isolation Exceptions. Also, the `_tags` should be removed. Only `tags` are returned in the response, and keys should be sorted in alphabetically ascending order.

```suggestion
{
 "_version": "WzcsMV0=",
  "created_at": "2020-07-13T09:33:46.187Z",
  "created_by": "elastic",
  "description": "Endpoint Security Blocklists List",
  "id": "f320c070-c4eb-11ea-80bb-11861bae2798",
  "immutable": false,
  "list_id": "{endpoint-artifact-list-id}",
  "name": "Endpoint Security Blocklists List",
  "namespace_type": "agnostic",
  "os_types": [],
  "tags": [],
  "tie_breaker_id": "2c08d5a5-2ecc-4d5a-acfb-0a367f25b3f3",
  "type": "endpoint_blocklists",
  "updated_at": "2020-07-13T09:33:46.359Z",
  "updated_by": "elastic",
  "version": 1
}
```